src/OversetSimulation.h

```diff
@@ -32,6 +32,10 @@ class OversetSimulation
     int m_last_timestep{0};
     //! Flag indicating whether initialization tasks have been performed
     bool m_initialized{false};
+    //! Flag indicating if complementary comms have been initialized
+    bool m_comp_comm_initialized{false};
```

Comment:
can you call this `m_complementary_comm_etc` ? just so the name is a bit more explicit

Reply:
Will do.

Reply:
This has been changed to `m_complementary_comm_initialized`.